OWML.ModHelper/Logging/ModConsole.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System;
 using OWML.Common;
 
-namespace OWML.ModHelper
+namespace OWML.ModHelper.Logging
```

Comment:
pretty sure that compatibility-breaking

Reply:
did you test it?

Reply:
The sample mods worked fine

Reply:
Since you just do using OWML.ModHelper, that already includes ModHelper.Logging

Reply:
seems to work fine with pre-compiled mods

Reply:
the specific case that breaks is: `ModConsole.OnConsole` (static event)

Reply:
If it doesn't break then we'd want the logging stuff extracted to a Logging project. @Raicuparta worked on it

Reply:
@misternebula pelase revert this namespace change for now. It breaks your own Console mod 👴 We'll save it for OWML 1.0